README.md

```diff
@@ -1 +1,208 @@
 # wavefront-csharp-sdk
+
+This package provides support for sending metrics, histograms and opentracing spans to Wavefront via proxy or direct ingestion.
+
+## Usage
+
+### Send data to Wavefront via Proxy
+```
+/*
+ * Assume you have a running Wavefront proxy listening on at least one of 
+ * metrics/direct-distribution/tracing ports and you know the proxy hostname
+ */
+var builder = new WavefrontProxyClient.Builder(proxyHost);
+
+/* set this (Example - 2878) if you want to send metrics to Wavefront */
+builder.MetricsPort(metricsPort);
+
+/* set this (Example - 40,000) if you want to send histograms to Wavefront */
+builder.DistributionPort(distributionPort);
+
+/* set this (Example - 30,000) if you want to send opentracing spans to Wavefront */
+builder.TracingPort(tracingPort);
+
+/* set this if you want to change the default flush interval of 5 seconds */
+builder.FlushIntervalSeconds(2);
```

Comment:
Let's encourage users to set it higher not lower :) Maybe 30 seconds in this example.